### ISSUE TITLE
Renewal

### ISSUE THREAD

**Issue by chrismask** [markrjones/listeo-package-handler#3]
The reverse pageflow of this plugin is super useful for adding new listings but when trying to renew subscription or change package, its counter-intuitive. Especially when pressing change package button, having to go through all this steps when you just want to change package so you get the new options on the next step in weird. I suggest for (/?action=renew*) URL, the pageflow shall not change from the default one.

**Comment by markrjones** [comment]
Hi Christos. Renewal wasn't considered when writing the plugin so I will have a look at what can be done to streamline this process.

**Comment by chrismask** [comment]
Yes, I know it wasnt proposed before, but after testing and playing a bit with the plugin I think it should be added to help the users. The (/?action=renew*) URL comes by default, so I think it would be fairly easy to bypass the new pageflow and just use the default one.

**Comment by markrjones** [comment]
Agreed, it would be a very good addition.
